R/run_stress_test.R

```diff
@@ -366,3 +346,52 @@ iteration_sequence <- function(args_list) {
     return(seq_along(args_list[[iter_var]]))
   }
 }
+
+st_read <- function(input_path_project_specific, asset_type) {
+  path <- pacta_results_path(input_path_project_specific, asset_type)
+
+  out <- list(
+    pacta_results = read_pacta_results(path)
+  )
+
+  return(out)
+}
+
+st_process <- function(data, asset_type) {
```

Comment:
Quick FYI so you know where this function will develop to. Currently a lot of, potentially configureable, variables are hardcoded.
In theshort term scenario_geography filter and c(baseline_scenario_lookup, shock_scenario_lookup) will be configurable and no longer hardcoded, several other variables might follow.